api/frontend/frontend/apps/api_docs/v1/docs/finder_data_structures.apibp

```diff
@@ -25,7 +25,8 @@
 
 ## PostcodeLocation (object)
 + type: `Feature` (string)
-+ properties: (object, nullable)
++ properties: (object)
+    + postcode: `SW1A 1AA` (string) - The postcode for this location
```

Comment:
I think I would phrase this slightly differently.. every point doesn't get its own postcode.
It is more like the postcode that this point is the centroid of, right?